uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/tripallocation/TripAllocationPresenter.kt

```diff
@@ -62,11 +71,9 @@ class TripAllocationPresenter(view: TripAllocationMVP.View,
     }
 
     private fun tripAllocated(trip: TripInfo) {
-        tripDetailsObservable?.apply {
-            tripDetailsObserver?.let {
-                unsubscribe(it)
-            }
-        }
+        showAllocationAlert = false
+        showAllocationAlert = false
```

Comment:
Is there a reason this is duplicated twice? "showAllocationAlert = false"

Reply:
Deleted